src/vmm/src/device_manager/legacy.rs

```diff
@@ -113,23 +114,39 @@ impl PortIODeviceManager {
         self.io_bus
             .insert(self.i8042.clone(), 0x060, 0x5)
             .map_err(Error::BusError)?;
+
+        macro_rules! register_irqfd_evt {
+            ($evt: ident, $index: expr) => {{
+                vm_fd
+                    .register_irqfd(&self.$evt, $index)
+                    .map_err(|e| Error::EventFd(std::io::Error::from_raw_os_error(e.errno())))?;
+            }};
+        }
+        register_irqfd_evt!(com_evt_1_3, 4);
+        register_irqfd_evt!(com_evt_2_4, 3);
+        register_irqfd_evt!(kbd_evt, 1);
```

Comment:
There are 10 loc here, if we get rid of the macro there will be 9. Can we remove the macro? WDYT ?

Reply:
done